Readme.md

```diff
@@ -1,7 +1,7 @@
 Axinom CPIX library
 ===================
 
-A .NET library for working with CPIX (Content Protection Information Exchange) documents, as defined by [DASH Industry Forum](http://dashif.org/guidelines/). This library implements CPIX version 2.1.
+A .NET library for working with CPIX (Content Protection Information Exchange) documents, as defined by [DASH Industry Forum](http://dashif.org/guidelines/). This library implements CPIX version 2.2.
```

Comment:
Isn't it .NET standard? If yes, perhaps valuable to mention. People associated .NET with the old stuff.